model/order.go

```diff
@@ -58,3 +60,13 @@ func (o Order) String() string {
 	return fmt.Sprintf("[%s] %s %s | ID: %d, Type: %s, %f x $%f (~$%.f)",
 		o.Status, o.Side, o.Pair, o.ID, o.Type, o.Quantity, o.Price, o.Quantity*o.Price)
 }
+
+func (o Order) ToPosition(leverage int) *Position {
+	return &Position{
+		Pair:       o.Pair,
+		Side:       o.Side,
+		Quantity:   o.Quantity,
+		EntryPrice: o.Price,
+		Leverage:   leverage,
+	}
+}
```

Comment:
This is not correct, a position can be a group of orders, and it is not converted directly